kinyarwanda/1.1994 Genocide against tutsi in Rwanda/1994-genocide-in-Rwanda

```diff
@@ -0,0 +1 @@
+In 1994, Rwanda faced a terrible genocide with a loss of more than 1 millions of lives of tustsi.
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.